### ISSUE TITLE
cryptic reporting of parse errors

### ISSUE THREAD

**Issue by glenjamin** [babel/babel-eslint#3]
```js
{ , res }
```

```
define-no-use.js
  0:0  error  2)

✖ 1 problem (1 error, 0 warnings)
```
